apps/docs/src/guide/wallets/connectors.md

```diff
@@ -1,8 +1,424 @@
+
+It will return a promise that will resolve to the current network (see [`Network`](../../api/Account/index.md#network)).
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-currentNetwork{ts:line-numbers}
+
+#### `selectNetwork`
+
+The `selectNetwork` method requests the user to select a network for the current connection.
+
+It requires a single argument:
+
+- `network` ([`Network`](../../api/Account/index.md#network))
+
+It will return a promise that resolves to `true` if the network is successfully selected; otherwise, it will return `false`.
+
+It should throw an error if the network is not available or the network does _not_ exist.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-selectNetwork{ts:line-numbers}
+
+#### `addABI`
+
+The `addABI` method adds ABI information about a contract to the connector. This operation does not require an authorized connection.
+
+It requires two arguments:
+
+- `contractId` (`string`)
+- `abi` ([`FuelABI`](../../api/Account/index.md#fuelabi)).
+
+It will return a promise that will resolve to `true` if the ABI is successfully added; otherwise `false`.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-addABI{ts:line-numbers}
+
+#### `getABI`
+
+The `getABI` method is used to get the ABI information that is sorted about a contract.
+
+It requires a single argument:
+
+- `contractId` (`string`)
+
+Returns a promise that resolves to the ABI information (as a [`FuelABI`](../../api/Account/index.md#fuelabi)) or `null` if the data is unavailable.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-getABI{ts:line-numbers}
+
+#### `hasABI`
+
+The `hasABI` method checks if the ABI information is available for a contract.
+
+It requires a single argument:
+
+- `contractId` (`string`)
+
+Returns a promise that resolves to `true` if the ABI information is available; otherwise `false`.
+
+<<< @/../../../packages/account/src/connectors/fuel-connector.ts#fuel-connector-method-hasABI{ts:line-numbers}
+
+## Connectors Manager
+
+The TS SDK exports the `Fuel` class, which serves as the connectors manager. This class provides the interface for interacting with the TS SDK and the broader Fuel ecosystem.
+
+It can be instantiated as follows:
+
+<<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-instantiation-1{ts:line-numbers}
+
+### Options
+
+Several options can be passed to the `Fuel` connector manager:
+
+- [`connectors`](#connectors)
+- [`storage`](#storage)
+- [`targetObject`](#targetobject)
+
+#### `connectors`
+
+The `connectors` option provides a list of connectors with which the `Fuel` connector manager can interact. The manager interacts with the connectors, which in turn handle communication with the respective wallet. You can find a list of all the connectors in our [`FuelLabs/fuel-connectors`](https://github.com/FuelLabs/fuel-connectors).
```

Comment:
@petertonysmith94 create an issue:
- requesting for a list of all the currently supported connectors and their related wallet install links.
- update this line + link to reference that page.

> You can find a list of all the connectors in our [`FuelLabs/fuel-connectors`](https://github.com/FuelLabs/fuel-connectors).

Reply:
Is the intention here to create an issue and handle this on a separate PR? I'm not seeing the issue

Reply:
- https://github.com/FuelLabs/fuels-ts/issues/2834